test/remote/gateways/remote_rapyd_test.rb

```diff
@@ -196,6 +196,31 @@ def test_failed_verify
     assert_equal 'Do Not Honor', response.message
   end
 
+  def test_successful_store_and_unstore
+    store = @gateway.store(@credit_card, @options)
+    assert_success store
+    assert customer_id = store.params.dig('data', 'id')
+
```

Comment:
How would you feel about expanding this test to check that a purchase call with the stored card is successful, and to further expect a failed purchase attempt after the card is unstored?

Reply:
That sounds like a good idea. I'll add that test.

Reply:
I'm trying to figure out the behavior here, but it looks like the payment method needs to go through 3DS authorization in order to be used. Seems odd since I'm using [what I thought was] a non-3DS payment method (`us_visa_card`).

If we want to test this, it'll have to be done manually. Drop these 2 lines below the `store` call and then use the `redirect_url` to go to the browser and "authorize" the card.

```ruby
store.params.dig('data', 'payment_methods', 'data', 0, 'redirect_url')
sleep(20) # adjust sleep as needed
```